conformance/README.md

```diff
@@ -35,6 +35,7 @@ uninstall-nkg                  Uninstall NKG on configured kind cluster
 | ------------- | ------------- | ------------- |
 | TAG | latest  | The tag for the conformance test image |
 | PREFIX | conformance-test-runner | The prefix for the conformance test image |
+| BUILD_NKG | true | Flag to indicate if the local NKG image needs to be built |
```

Comment:
this variable looks a bit a hacky. is it possible to refactor targets so that we don't need it?
for example: split build and load and make them not depend on each other